code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -139,40 +139,44 @@
 	color = rgb(255,251,0)
 
 /obj/structure/bed/chair/withwheels
+	var/delay = 10
 	var/cooldown = 0
+	var/reverse_direction = 1
 
-/obj/structure/bed/chair/withwheels/relaymove(mob/user, direction)
+/obj/structure/bed/chair/withwheels/relaymove(mob/user, direction, delay=10)
 	if((!Process_Spacemove(direction)) || (!has_gravity(src.loc)) || (cooldown) || user.stat || user.stunned || user.weakened || user.paralysis || (user.restrained()))
 		return
 	step(src, direction)
 	if(buckled_mob)
-		buckled_mob.dir = dir
-		switch(buckled_mob.dir)
-			if(NORTH)
-				buckled_mob.dir = SOUTH
-			if(WEST)
-				buckled_mob.dir = EAST
-			if(SOUTH)
-				buckled_mob.dir = NORTH
-			if(EAST)
-				buckled_mob.dir = WEST
+		buckled_mob.dir = direction
+		if(src.reverse_direction)
+			switch(direction)
+				if(NORTH)
+					buckled_mob.dir = SOUTH
+				if(WEST)
+					buckled_mob.dir = EAST
+				if(SOUTH)
+					buckled_mob.dir = NORTH
+				if(EAST)
+					buckled_mob.dir = WEST
 		dir = buckled_mob.dir
 	handle_rotation()
 	handle_layer()
 	cooldown = 1
-	spawn(10)
+	spawn(src.delay ? src.delay : delay)
```

Comment:
you've set a default value to all "withwheels" chairs, var/delay = 10. If your child won't have a different value,it'll inherit the parent's one,therefore you can just do spawn(delay)